src/mds/FSMap.h

```diff
@@ -576,8 +576,8 @@ class FSMap {
   uint64_t next_filesystem_id = FS_CLUSTER_ID_ANONYMOUS + 1;
   fs_cluster_id_t legacy_client_fscid = FS_CLUSTER_ID_NONE;
   CompatSet compat;
-  bool enable_multiple = false;
-  bool ever_enabled_multiple = false; // < the cluster had multiple MDSes enabled once
+  bool enable_multiple = true;
+  bool ever_enabled_multiple = true; // < the cluster had multiple FS enabled once
```

Comment:
Is the ever_enabled_multiple flag actually used for anything? It looks like it gets set in the FSMap, but I don't see where anything actually uses it.

Reply:
We have historically used this to know whether the admin used a feature and then turned it off. This was particularly important for knowing if snapshots were turned on in the past. So this bool was added for the multifs feature too.